server/profiles/skhan/config-build.properties

```diff
@@ -0,0 +1,7 @@
+oxTrust.debug=true
```

Comment:
Should we add test files to the repo or should we add them to the gitignore?
@yurem can you please assist on this question ?